arch/TriCore/TriCoreDisassembler.c

```diff
@@ -699,10 +756,24 @@ static DecodeStatus DecodeRRInstruction(MCInst *Inst, unsigned Insn,
 	}
 
 	if (desc->NumOperands > 1) {
-		status = DecodeRegisterClass(Inst, s1, &desc->OpInfo[1],
-					     Decoder);
-		if (status != MCDisassembler_Success)
-			return status;
+		if (desc->OpInfo[0].OperandType == MCOI_OPERAND_REGISTER) {
+			switch (MCInst_getOpcode(Inst)) {
+			case TRICORE_ABSS_rr:
+			case TRICORE_ABSS_H_rr:
+			case TRICORE_ABS_H_rr:
+			case TRICORE_ABS_B_rr:
+			case TRICORE_ABS_rr: {
```

Comment:
I checked and found that you still haven't fully fixed it here.
Can you please double-check the following instructions?

### `d, s2[, s1[, n]]`
ADDSC.A
ADDSC.AT
MOV.A
MOV.AA
MOV.D
CRC32.B
CRC32B.W
CRC32L.W

### `a`
UPDFL

Reply:
They have been correct before. Not sure why. These are the insns I sent to capstone as well as GNU objdump and they agree.
```
CODE = b'\x01\x23\x01\x16'  # addsc.a %a1,%a2,%d3,1
CODE += b'\x01\x34\x20\x26' # addsc.at %a2,%a3,%d4
CODE += b'\x01\x20\x30\x16' # mov.a %a1,%d2
CODE += b'\x01\x30\x00\x20' # mov.aa %a2,%a3
CODE += b'\x01\xb0\xc0\xa4' # mov.d %d10,%a11
CODE += b'\x4b\xbc\x60\xa0' # crc32.b %d10,%d11,%d12
CODE += b'\x4b\xbc\x30\xa0' # crc32 %d10,%d11,%d12
CODE += b'\x4b\xbc\x70\xa0' # crc32l.w %d10,%d11,%d12
CODE += b'\x4b\x0a\xc1\xa0' # updfl %d10
```

Reply:
I remembered that we can also change the order of operands in the td file.